changelog.txt

```diff
@@ -1,5 +1,8 @@
 *** WooCommerce Shipping & Tax Changelog ***
 
+= 1.25.17 - xxxx-xx-xx =
```

Comment:
TIL we don't need the year

Reply:
Just double checked, `xxxx` should work for year as well!